### ISSUE TITLE
Awaited statements should be preceded and followed by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#333]
An awaited call should be followed by a blank line if the following line contains a non-awaited expression.
The reason is ease of reading.

Following should report a violation:
```c#
await DoStuffAsync();
await DoMoreStuffAsync();
var x = 42;
var y = "something";
var z = Guid.NewGuid();
```

While following should **not** report a violation:
```c#
await DoStuffAsync();
await DoMoreStuffAsync();

var x = 42;
var y = "something";
var z = Guid.NewGuid();
```

**Comment by RalfKoban** [comment]
An awaited call should also be preceded by a blank line if the preceding line contains a non-awaited expression.
The reason is ease of reading.
